packages/kit/src/exports/index.js

```diff
@@ -72,3 +73,23 @@ export function text(body, init) {
 export function fail(status, data) {
 	return new ActionFailure(status, data);
 }
+
+/**
+ * Populate a route ID with params to produce relative path.
+ * @example
+ * ```js
+ * resolvePath(
+ *   `/blog/[slug]/[...somethingElse]`,
+ *   {
+ *     slug: 'hello-world',
+ *     somethingElse: 'something/else'
+ *   }
+ * ); // `/blog/hello-world/something/else`
+ * ```
+ * @param {string} id
+ * @param {Record<string, string>} params
+ * @returns {string}
+ */
+export function resolvePath(id, params) {
```

Comment:
Since we call it route id in other places I think it would make sense to reuse that nomenclature here
```suggestion
export function resolveRoute(id, params) {
```
Alternatively even `resolveRouteId`. 

Thoughts?

Reply:
I think `resolvePath` is correct — we're resolving a path from a route ID + some parameters.

That said, `resolvePath` sounds like `path.resolve`, i.e. you might expect to see `resolvePath(base, relative)`. I don't know if there's a good name for this that doesn't sound clunky and un-SvelteKit-like.

Reply:
Yeah, that was my reasoning as well, Rich -- I think the current name is a good mix of what it does and brevity, and the example is pretty darn clear